_posts/2022-12-14-#26631.md

```diff
@@ -10,17 +10,117 @@ status: upcoming
 commit: 
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+### concepts
+
+- Each bitcoin transaction output carries a specific
+  [value](https://github.com/bitcoin/bitcoin/blob/6061eb6564105ad54703a7cf3282590d0e1a7f28/src/primitives/transaction.h#L159).
+  Bitcoin Core defines _dust_ as an output whose value is less than
+  what it would cost to spend this output.
+
+- An important goal of the Bitcoin network is to keep the resource costs
+  for running a fully-validating node to a minimum. One way to reduce
+  the storage requirement is to keep the size of the UTXO set small.
+
+- If it were possible for an attacker, or a careless user, to create
+  many tiny-value UTXOs, the UTXO set could be increased dramatically,
+  because whoever is able to spend these UTXOs (and thus remove them
```

Comment:
I would call it a problem with the wallet rather than the user. It's also still 100% possible
```suggestion
- It would be inexpensive for an attacker, or a careless wallet, to create
  many tiny-value UTXOs, bloating the UTXO set. Additionally,
  whoever is able to spend these UTXOs (and thus remove them
```